supply-chain-security/compromises/README.md

```diff
@@ -8,16 +8,26 @@ can better understand the patterns and develop best practices and tools.
 
 We welcome additions to this catalog by 
 [filing an issue](https://github.com/cncf/sig-security/issues/new/choose) or
-[github pull request](https://github.com/cncf/sig-security)
+submitting a [pull request](https://github.com/cncf/sig-security) to our repo.
+
+Additions aren't intended to be a statement on the security stance and/or
+practices of any given project, or of open source as a whole. If anything,
+many of the stories showcase rapid response, evolving security practices in
+the community and an open and transparent approach to disclosure.
 
 
 | Name              | Year               | Type of compromise    | Link        |
 | ----------------- | ------------------ | ------------------    | ----------- |
 | [electron-native-notify](2019/electron-native-notify.md) | 2019 | Source Code Compromise | [1](https://blog.npmjs.org/post/185397814280/plot-to-steal-cryptocurrency-foiled-by-the-npm), [2](https://komodoplatform.com/update-agama-vulnerability/)|
+| [PyPI typosquatting](2019/pypi.md) | 2019 | Naming attack | [1](https://blog.reversinglabs.com/blog/suppy-chain-malware-detecting-malware-in-package-manager-repositories) |
+| [ROS build farm compromise](2019/ros.md) | 2019 | Build infrastructure | [1](https://discourse.ros.org/t/security-issue-on-ros-build-farm/9342/8), [2](https://discourse.ros.org/t/new-gpg-keys-deployed-for-packages-ros-org/9454) |
 | [ShadowHammer](2019/shadowhammer.md) | 2019 | Multiple steps | [1](https://www.csoonline.com/article/3384259/asus-users-fall-victim-to-supply-chain-attack-through-backdoored-update.html), [2](https://securelist.com/operation-shadowhammer/89992/) |
 | [PEAR Breach](2019/pear.md) | 2019 | Publishing Infrastructure | [1](https://blog.dcso.de/php-pear-software-supply-chain-attack/), [2](https://thehackernews.com/2019/01/php-pear-hacked.html) |
+| [Canonical's GitHub org compromised](2019/canonical-github.md) | 2019 | Build and publishing infrastructure | [1](https://www.zdnet.com/article/canonical-github-account-hacked-ubuntu-source-code-safe/) |
 | [Dofoil](2018/dofoil.md) | 2018 | Publishing Infrastructure | [1](https://www.zdnet.com/article/windows-attack-poisoned-bittorrent-client-set-off-huge-dofoil-outbreak-says-microsoft/) |
 | [Operation Red](2018/operation-red.md) | 2018 | Publishing Infrastructure | [1](https://blog.trendmicro.com/trendlabs-security-intelligence/supply-chain-attack-operation-red-signature-targets-south-korean-organizations/) |
+| [RCE in go get -u](2018/gogetu.md) | 2018 | Tooling attack | [1](https://github.com/golang/go/issues/29230), [2](https://staaldraad.github.io/post/2019-03-28-go-get-vuln/) |
+| [acroread compromised in AUR](2018/aur.md) | 2018 | Naming attack | [1](ttps://lists.archlinux.org/pipermail/aur-general/2018-July/034152.html), [2](https://www.bleepingcomputer.com/news/security/malware-found-in-arch-linux-aur-package-repository/) |
```

Comment:
Change `(Naming attack)` to either`(Orphaned Package Takeover)` or '(Malicious Adoption)' for consistency with previous comment regarding acroread.

Reply:
@SantiagoTorres do you concur?

Reply:
Malicious Maintainer - definition already added updating here.